app/scripts/lib/createOriginThrottlingMiddleware.ts

```diff
@@ -17,15 +17,16 @@ export const REJECTION_THRESHOLD_IN_MS = 30000;
 export const BLOCKING_THRESHOLD_IN_MS = 60000;
 
 export const BLOCKABLE_METHODS: Set<string> = new Set([
+  MESSAGE_TYPE.ADD_ETHEREUM_CHAIN,
   MESSAGE_TYPE.ETH_SEND_TRANSACTION,
   MESSAGE_TYPE.ETH_SIGN_TYPED_DATA,
   MESSAGE_TYPE.ETH_SIGN_TYPED_DATA_V1,
   MESSAGE_TYPE.ETH_SIGN_TYPED_DATA_V3,
   MESSAGE_TYPE.ETH_SIGN_TYPED_DATA_V4,
   MESSAGE_TYPE.PERSONAL_SIGN,
-  MESSAGE_TYPE.WATCH_ASSET,
-  MESSAGE_TYPE.ADD_ETHEREUM_CHAIN,
   MESSAGE_TYPE.SWITCH_ETHEREUM_CHAIN,
+  MESSAGE_TYPE.WALLET_SEND_CALLS,
```

Comment:
Perhaps out of scope, but have we tested if the warning triggers if the user keeps rejecting the upgrade rather than the transaction itself?

Reply:
when I tested, I repeatedly canceled to see the warning 👍🏼 

@matthewwalsh0